docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Clustering/KMeans.cs

```diff
@@ -10,38 +10,49 @@ public static class KMeans
     {
         public static void Example()
         {
-            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
-            // as a catalog of available operations and as the source of randomness.
-            // Setting the seed to a fixed number in this example to make outputs deterministic.
+            // Create a new context for ML.NET operations. It can be used for
+            // exception tracking and logging, as a catalog of available operations
+            // and as the source of randomness. Setting the seed to a fixed number
+            // in this example to make outputs deterministic.
             var mlContext = new MLContext(seed: 0);
 
-            // Convert the list of data points to an IDataView object, which is consumable by ML.NET API.
+            // Convert the list of data points to an IDataView object, which is
+            // consumable by ML.NET API.
```

Comment:
Is this really an IDataView?

Reply:
You are right this comment should be updated.

---
In reply to: [278766667](https://github.com/dotnet/machinelearning/pull/3595#discussion_r278766667) [](ancestors = 278766667)